src/Components/Resource/ResourceDetailsUpdate.tsx

```diff
@@ -67,13 +64,9 @@ const initialState = {
 
 export const ResourceDetailsUpdate = (props: resourceProps) => {
   const { goBack } = useAppHistory();
-  const dispatchAction: any = useDispatch();
   const [qParams, _] = useQueryParams();
   const [isLoading, setIsLoading] = useState(true);
-  const [assignedQuantity, setAssignedQuantity] = useState(0);
-  const [requestTitle, setRequestTitle] = useState("");
-  const [assignedUser, SetAssignedUser] = useState(null);
-  const [assignedUserLoading, setAssignedUserLoading] = useState(false);
+  const [assignedUser, SetAssignedUser] = useState<UserModel>();
```

Comment:
You can instead take the data from the useQuery and do this instead of having a `useState`

```jsx
const assignedUser = data?.count && data?.results[0]
```

Reply:
removing the state in the view (frontend) the same assinged to is displayed even when changing the option  
clear and changing the option is not working the same assigned name is displayed